miasm2/core/asmblock.py

```diff
@@ -1325,6 +1325,27 @@ def asm_resolve_final(mnemo, blocks, symbol_pool, dst_interval=None):
     return patches
 
 
+class DeprecatedJobDone(object):
```

Comment:
Why don't you use a property, with a getter and a setter? (returning an actual set) 
If a user is doing a `mis.job_done = set() `, he will not see the warning